packages/lit-dev-content/site/blog/2021-09-21-announcing-lit-2.md

```diff
@@ -75,7 +75,7 @@ For a full list of changes, see the [changelog](https://github.com/lit/lit/blob/
 
 ## Get started with Lit 2
 
-Want to try out Lit 2? Visit [lit.dev](https://lit.dev/docs/getting-started/) to get started. The site features an [interactive tutorial](https://lit.dev/tutorial/) to get you started with Lit 2, an interactive [playground](https://lit.dev/playground/) for live coding, guides, and API docs.
+Want to try out Lit 2? Visit [lit.dev](https://lit.dev/docs/getting-started/) to get started. The site features an [interactive tutorial](https://lit.dev/tutorials/intro-to-lit) to get you started with Lit 2, an interactive [playground](https://lit.dev/playground/) for live coding, guides, and API docs.
```

Comment:
Maybe worth re-wording that the site features multiple tutorials, or that there is an Into to Lit tutorial which is great for getting started.